Automattic/wp-calypso
# Jetpack Manage: Move both 'Issue license' and 'Payment Methods' page components within the Primary folder.

Earlier, we discussed moving all Page components within the Primary folder for folder structure consistency. This PR should not affect any functionality.

## Proposed Changes

* Move the `partner-portal/issue-license-v2` folder to the `partner-portal/primary` folder while renaming the folder to `issue-license`.
* Move the `partner-portal/payment-method-v2` folder to the `partner-portal/primary` folder.

## Testing Instructions
Since these changes are made specifically for agencies, you must set yourself(partner) as an agency - 2c49b-pb.

* Use the Jetpack cloud live link below and Go to the Issue license page (`partner-portal/issue-license`)
* Confirm that the page works perfectly and nothing has been broken.
* Next, Go to the Payment Method page (`partner-portal/payment-methods`)
* COnfirm that the page works perfectly.

## Pre-merge Checklist
